### PR TITLE
Fix `Custom` username type validation visibility issue

### PR DESCRIPTION
**Pull request by jcgarciaa**
### Purpose
With this update the configuration section of the username validation option won't be hidden when clicking the `Custom` radio button immediately after the page load.

### Related Issues
- None

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx [hunk 1]
```diff
@@ -110,6 +106,13 @@ export const UsernameValidationEditPage: FunctionComponent<UsernameValidationEdi
 
     }, [ initialFormValues ]);
 
+    useEffect(() => {
+        if (currentValues !== undefined) {
```
Comment (Achintha444) [inline apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx:110]:
```suggestion
        if (currentValues) {
```

File: apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx [hunk 2]
```diff
@@ -98,18 +97,22 @@ export const UsernameValidationEditPage: FunctionComponent<UsernameValidationEdi
...
-        if (initialFormValues !== undefined) {
+        if (initialFormValues) {
             setCurrentValues({ ...initialFormValues });
         }
 
     }, [ initialFormValues ]);
 
+    useEffect(() => {
+        if (currentValues) {
+            setTimeout(setPageLoaded, 200, true);
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx:111]: IMO, setting an arbitrary timeout is not the correct way to solve this problem. AFAIU, the component is remounted during rendering the custom username validation config. We should find the RC for that remounting and fix it.

cc: @brionmario

Comment (brionmario) [reply apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx:111]: +1. Let's try to figure out why the issue is happening.

Comment (jcgarciaa) [reply apps/console/src/extensions/components/account-login/pages/username-validation-edit.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5217/commits/31442763afa9fa105ca1fc7c5878f2351708e9e0

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**